rich/console.py

```diff
@@ -40,7 +40,7 @@
 from .markup import render as render_markup
 from .measure import Measurement, measure_renderables
 from .pager import Pager, SystemPager
-from .pretty import is_expandable, Pretty
+from .pretty import Pretty, is_expandable
```

Comment:
Sorry this was automatically sorted due to my VSCode settings, I can revert this if you prefer